### PR TITLE
Fix the issue related to Add members to Channel endpoint

### PR DESCRIPTION
**Pull request by jujucoder**
https://linear.app/team-bevel/issue/BAC-74/fix-the-issue-related-to-add-members-to-channel-endpoint
i added a error handling  code block in the /org/{org_id}/rooms/{room_id}/members/{member_id} (post) 
 endpoint that add members to a room in the backend/endpoints/rooms.py file, to ensure that a member id must exist in the room_members of a public room to add a new member

### CODE REVIEW

File: backend/endpoints/rooms.py [hunk 1]
```diff
@@ -238,11 +238,14 @@ async def join_room(
...
+        raise HTTPException(
+                status_code=status.HTTP_401_UNAUTHORIZED,
+                detail="only existing members can add new members",
+            )
 
     if room["room_type"].upper() == RoomType.CHANNEL:
-        if room["is_private"] is True and (
-            member is None or member["role"].lower() != Role.ADMIN
-        ):
+        if room["is_private"] is True and (member["role"].lower() != Role.ADMIN):
```
Comment (mikengr) [inline backend/endpoints/rooms.py:248]: @jayjaytentacion what was the bug and how does this fix it?

**Review by mikengr — COMMENTED** [review]
@jayjaytentacion I left a comment

File: backend/endpoints/rooms.py (hunk 1 above)
Comment (jujucoder) [reply backend/endpoints/rooms.py:248]: The bug is If a non existing member_id or one that isn't a member of a room, is used to make a request to add  new member(s), it's adds new member(s)
Line 241 - line 245 is solving it

Comment (mikengr) [reply backend/endpoints/rooms.py:248]: Okay, makes sense. 
Is that all there is to this issue? What happened to the attached sub-tissues?

Comment (jujucoder) [reply backend/endpoints/rooms.py:248]: I tested them  and they where working fine

Comment (jujucoder) [reply backend/endpoints/rooms.py:248]: @mikengr

Comment (mikengr) [reply backend/endpoints/rooms.py:248]: @jayjaytentacion send me the endpoint where you tested for this -> 
[BAC-76](https://linear.app/team-bevel/issue/BAC-76/build-a-channel-delete-member-endpoint)

**Comment by FlorenceEgwu** [comment]
Looks good to me. 
But I need access to the linear board to view the linear issue. so as to give any helpful review

File: backend/endpoints/rooms.py (hunk 1 above)
Comment (jujucoder) [reply backend/endpoints/rooms.py:248]: https://chat.zuri.chat/docs#/rooms/remove_member_api_v1_org__org_id__rooms__room_id__members__member_id__patch

Comment (jujucoder) [reply backend/endpoints/rooms.py:248]: i created a channel, then i used  postman to and and remove members

Comment (jujucoder) [reply backend/endpoints/rooms.py:248]: @mikengr

**Comment by FlorenceEgwu** [comment]
just checked linear. you still have 3 sub issues to fix. I presume you be making another PR?

**Comment by jujucoder** [comment]
> just checked linear. you still have 3 sub issues to fix. I presume you be making another PR?

one endpoint handles the functionality of the two sub issues and it already exists, i tested it and it woking fine
@AI-fae

**Pull request merged**